src/Api/Projects.php

```diff
@@ -1704,6 +1704,31 @@ public function createProjectAccessToken($project_id, array $parameters = [])
         return $this->post($this->getProjectPath($project_id, 'access_tokens'), $resolver->resolve($parameters));
     }
 
+    /**
+     * @param int|string $project_id
+     * @param int        $token_id
+     * @param array      $parameters {
+     *
+     *     @var \DateTimeInterface $expires_at  the token expires at midnight UTC on that date
+     * }
+     *
+     * @return mixed
+     */
+    public function rotateProjectAccessToken($project_id, int $token_id, array $parameters = [])
```

Comment:
Crosslinking both (open) GitHub pull requests with the same method/functionality:
1. https://github.com/GitLabPHP/Client/pull/800/files#diff-160122af7affd3b85f6c02a3c1fcd7f20fbda356ab8bf3c89f71d0a56f68b31bR1725
2. https://github.com/GitLabPHP/Client/pull/808/files#diff-160122af7affd3b85f6c02a3c1fcd7f20fbda356ab8bf3c89f71d0a56f68b31bR1717